### PR TITLE
LCDproc: add a shutdown/reboot control menu. Issue #8196

### PR DESCRIPTION
**Pull request by vktg**
Redmine Issue: https://redmine.pfsense.org/issues/8196
Ready for review

add a control menu with the name 'pfsense' next to the 'Options' menu of LCDproc.
Currently there are only two options implemented: Reboot and shutdown.
Structure:
```
Options (original LCDproc)
pfsense
  ->Reboot
    ->No
    ->Yes
  ->Shutdown
    ->No
    ->Yes
```

This is fixed/resolved copy of the original PR https://github.com/pfsense/FreeBSD-ports/pull/376 by @c-schwamborn

### CODE REVIEW

**Comment by vktg** [comment]
Why we need to use printf() instead of '<br />' as everywhere in this code https://github.com/pfsense/FreeBSD-ports/pull/376/files#r130587889 ?

**Comment by rbgarga** [comment]
> Why we need to use printf() instead of `<br />` as everywhere in this code https://github.com/pfsense/FreeBSD-ports/pull/376/files#r130587889 ?

setHelt() is one of the functions considered by our gettext config to be translated and we don't want to have HTML tags mixed with strings for translation.  setHelp accepts printf() similar parameters so you can replace <br /> by `%1$s` and pass `'<br />'` as a parameter

**Comment by vktg** [comment]
> > Why we need to use printf() instead of `<br />` as everywhere in this code https://github.com/pfsense/FreeBSD-ports/pull/376/files#r130587889 ?
> 
> setHelt() is one of the functions considered by our gettext config to be translated and we don't want to have HTML tags mixed with strings for translation. setHelp accepts printf() similar parameters so you can replace
> by `%1$s` and pass `'<br />'` as a parameter

ok, fixed in all lcdproc_client.php

File: sysutils/pfSense-pkg-LCDproc/files/usr/local/pkg/lcdproc_client.php [hunk 1]
```diff
@@ -425,6 +440,14 @@ function outputled_enabled_CFontz633() {
 	}
 }
 
+function cmenu_enabled() {
+	global $config;
+	$lcdproc_config = $config['installedpackages']['lcdproc']['config'][0];
+	if(!isset($lcdproc_config['controlmenu']))
+		return false;
```
Comment (jim-p) [inline sysutils/pfSense-pkg-LCDproc/files/usr/local/pkg/lcdproc_client.php]: `if` body needs braces

Comment (vktg) [reply sysutils/pfSense-pkg-LCDproc/files/usr/local/pkg/lcdproc_client.php]: fixed

File: sysutils/pfSense-pkg-LCDproc/files/usr/local/pkg/lcdproc_client.php [hunk 2]
```diff
@@ -425,6 +440,15 @@ function outputled_enabled_CFontz633() {
 	}
 }
 
+function cmenu_enabled() {
+	global $config;
+	$lcdproc_config = $config['installedpackages']['lcdproc']['config'][0];
+	if(!isset($lcdproc_config['controlmenu'])) {
```
Comment (jim-p) [inline sysutils/pfSense-pkg-LCDproc/files/usr/local/pkg/lcdproc_client.php]: Needs a space between `if` and `(`

Comment (vktg) [reply sysutils/pfSense-pkg-LCDproc/files/usr/local/pkg/lcdproc_client.php]: fixed

**Pull request merged**